Update Frostcrag Spire Coblized

charabaruk opened the pull request:
Frostcrag Spire Cobl Alchemy Station is now Frostcrag Spire Coblized.

Checksum for version 1.0 has been replaced with that of 2.0 (version numbers are continuing from the original plugin).

Frostcrag Spire Coblized includes DLC Frostcrag Cobl Addons entirely, and replaces the functionality of Frostcrag Alchemy Sorter. Messages are added to all three plugins regarding this.

charabaruk committed:
Update Frostcrag Spire Coblized

Frostcrag Spire Cobl Alchemy Station is now Frostcrag Spire Coblized.

Checksum for version 1.0 has been replaced with that of 2.0 (version
numbers are continuing from the original plugin).

Frostcrag Spire Coblized includes DLC Frostcrag Cobl Addons entirely,
and replaces the functionality of Frostcrag Alchemy Sorter. Messages are
added to all three plugins regarding this.

charabaruk commented:
Apparently `version()` doesn't do greater-or-equals?

sibir-ine commented:
It does, you just missed the commas separating them & the version number strings.

charabaruk commented:
D'OH!!!! 🤦‍♂️

charabaruk committed:
Fix syntax error in version() conditions

sibir-ine commented on masterlist.yaml:
```diff
@@ -20453,16 +20453,29 @@ plugins:
+    msg:
+      - <<: *obsolete
+        condition: 'version("DLCFrostcragCoblAlchemyTable.esp", "2.0", <)'
+      - <<: *includesX
+        subs: [ 'DLC Frostcrag Cobl Addons' ]
+        condition: 'version("DLCFrostcragCoblAlchemyTable.esp", "2.0", >=) and active("DLCFrostcragCoblAddons.esp")'
+      - <<: *includesX
+        subs: [ 'Frostcrag Alchemy Sorter' ]
+        condition: 'version("DLCFrostcragCoblAlchemyTable.esp", "2.0", >=) and active("FrostcragAlchemySorter.esp")'
```
```suggestion
```

These are redundant with the `alreadyInX` messages which is preferred in this context.

sibir-ine commented on masterlist.yaml:
```diff
@@ -20453,16 +20453,29 @@ plugins:
         name: 'DLC Frostcrag Cobl Addons'
+    msg:
+      - <<: *alreadyInX
+        subs: [ '[Frostcrag Spire Coblized](https://www.nexusmods.com/oblivion/mods/52608/)' ]
+        condition: 'active("DLCFrostcragCoblAlchemyTable.esp")'
```
```suggestion
        subs: [ '**DLCFrostcragCoblAlchemyTable.esp**' ]
        condition: 'active("DLCFrostcragCoblAlchemyTable.esp") and version("DLCFrostcragCoblAlchemyTable.esp", "2.0", >=)'
```

If they already have the plugin, you don't need to link to it. It's also clearer what plugin contains this plugin if you reference it by filename. You could also add the version check here to avoid confusion with users of older versions.

sibir-ine commented on masterlist.yaml:
```diff
@@ -20511,6 +20524,9 @@ plugins:
         subs: [ '[Frostcrag Alchemy Sorter Updated](https://www.nexusmods.com/oblivion/mods/49969/)' ]
         condition: 'version("FrostcragAlchemySorter.esp", "1.0", <)'
+      - <<: *alreadyInX
+        subs: [ '[Frostcrag Spire Coblized](https://www.nexusmods.com/oblivion/mods/52608/)' ]
+        condition: 'active("DLCFrostcragCoblAlchemyTable.esp")'
```
```suggestion
        subs: [ '**DLCFrostcragCoblAlchemyTable.esp**' ]
        condition: 'active("DLCFrostcragCoblAlchemyTable.esp") and version("DLCFrostcragCoblAlchemyTable.esp", "2.0", >=)'
```

Same as above. This message should also be sorted above the `patchUpdateAvailable` message (I can't do it in a suggestion).

charabaruk committed:
Tidy messages for Frostcrag Spire Coblized

Remove `includesX` messages as redundant, as `alreadyInX` on the
subsumed plugins is preferred. Update those `alreadyInX` messages as
well, to ensure they only appear if v2.0+ of Frostcrag Spire Coblized is
installed.

charabaruk commented:
Changes made.

sibir-ine reviewed (approved):
Thanks!
